CHANGELOG.md

```diff
@@ -1,3 +1,20 @@
+## 14.0.1
+
+_released `2023-05-22`_
+
+### cli
+
+- :scroll: Fix missing `key` property in the `.zapierapprc` file after running the `zapier convert` command
```

Comment:
We typically provide a link to the PR for each item in the CHANGELOG.
```suggestion
- :scroll: Fix missing `key` property in the `.zapierapprc` file after running the `zapier convert` command [#640](https://github.com/zapier/zapier-platform/pull/640)
```

Reply:
Oops, that's right! I've been playing around with auto-generating these and at this point I've forgotten how they are supposed to look like 😵 Will add this 🙏🏻